mysql-test/lib/My/Platform.pm

```diff
@@ -70,6 +70,14 @@ BEGIN {
   }
 }
 
+BEGIN {
```

Comment:
@vuvova I will need at least this change to properly support `--parallel` on macOS because we need to fetch the CPU information differently on that platform than we do on Linux or Windows.  Is there any reason to keep this patch back at this point?

Reply:
Cannot think of any. You've created a separate MDEV for this, I recall and @grooverdan wrote "ok to push into 10.5". I concur, please, feel free to push it, and preferably in 10.5.